notebooks/hidden_features/HiddenFeatures.ipynb

```diff
@@ -26,40 +26,22 @@
     "from certifai.scanner.builder import (CertifaiScanBuilder, CertifaiPredictorWrapper, CertifaiModel, CertifaiModelMetric,\n",
     "                                      CertifaiDataset, CertifaiGroupingFeature, CertifaiDatasetSource,\n",
     "                                      CertifaiPredictionTask, CertifaiTaskOutcomes, CertifaiOutcomeValue)\n",
-    "from certifai.scanner.report_utils import scores, construct_scores_dataframe"
+    "from certifai.scanner.report_utils import scores, construct_scores_dataframe\n",
+    "from certifai.common.utils.encoding import CatEncoder"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 2,
    "metadata": {},
```

Comment:
typo in header "Firstly we'll build a model that does not have access to the protected features and see to **waht** degree" -> waht -> what